### PR TITLE
[ADI] Add comments and thread list endpoints

### PR DESCRIPTION
**Pull request by illusionOfParadise**

### CODE REVIEW

File: features/api/ideas_workshop_comments.feature [hunk 1]
```diff
@@ -0,0 +1,92 @@
+@api
+Feature:
+  In order to see comments
+  As a user
+  I should be able to access API threads abd thread comments
```
Comment (thomasbisignani) [inline features/api/ideas_workshop_comments.feature]: `abd` => `and`

File: src/DataFixtures/ORM/LoadIdeaThreadData.php [hunk 2]
```diff
@@ -36,9 +39,32 @@ public function load(ObjectManager $manager)
         );
         $this->setReference('thread-aq-compare', $threadAQCompareAdherent5);
 
+        $threadRefused = new Thread(
+            'Une disscussion refusée.',
```
Comment (thomasbisignani) [inline src/DataFixtures/ORM/LoadIdeaThreadData.php]: `disscussion` => `discussion`

File: features/api/ideas_workshop_comments.feature (hunk 1 above)
Comment (illusionOfParadise) [reply features/api/ideas_workshop_comments.feature]: Done.

File: src/DataFixtures/ORM/LoadIdeaThreadData.php (hunk 2 above)
Comment (illusionOfParadise) [reply src/DataFixtures/ORM/LoadIdeaThreadData.php]: Done.

File: src/Entity/IdeasWorkshop/Idea.php [hunk 3]
```diff
@@ -50,12 +51,17 @@
...
 {
     use EntityIdentityTrait;
     use EntityTimestampableTrait;
     use EntityNameSlugTrait;
 
+    public static function getVisibleStatuses(): array
+    {
+        return IdeaStatusEnum::VISIBLE_STATUSES;
+    }
+
```
Comment (Remg) [inline src/Entity/IdeasWorkshop/Idea.php]: methods should be declared after constants

File: src/Entity/IdeasWorkshop/ThreadComment.php [hunk 4]
```diff
@@ -3,60 +3,53 @@
...
+ *         "normalization_context": {
+ *             "groups": {"thread_comment_read"}
+ *         },
+ *         "filters": {"threadComment.thread"},
+ *         "order": {"createdAt": "ASC"}
+ *     },
+ *     collectionOperations={"get"},
+ *     itemOperations={"get"},
+ * )
+ * @ApiFilter(SearchFilter::class, properties={"thread": "exact"})
```
Comment (ottaviano) [inline src/Entity/IdeasWorkshop/ThreadComment.php]: what about using sub resource instead `/threads/1/comments`?

File: features/api/ideas_workshop_idea_list.feature [hunk 5]
```diff
@@ -0,0 +1,217 @@
...
+    And the JSON should be equal to:
+    """
+    [
+        {
+            "theme": {
+                "name": "Armées et défense"
+            },
+            "category": {
+                "name": "Echelle Européenne",
+                "enabled": true
```
[the remark this answers is not in the document]
Comment (illusionOfParadise) [reply features/api/ideas_workshop_idea_list.feature]: No.

File: src/Entity/IdeasWorkshop/ThreadComment.php (hunk 4 above)
Comment (illusionOfParadise) [reply src/Entity/IdeasWorkshop/ThreadComment.php]: Done.

**Pull request merged**